src/database/models/student-topic-assessment-override.ts

```diff
@@ -31,6 +35,13 @@ export default class StudentTopicAssessmentOverride extends Model implements Stu
     public readonly createdAt!: Date;
     public readonly updatedAt!: Date;
 
+    static getOverrides = (obj: StudentTopicAssessmentOverrideOverridesInterface): StudentTopicAssessmentOverrideOverridesInterface => {
+        return _.pick(obj, 'duration', 'maxGradedAttemptsPerVersion', 'maxVersions', 'versionDelay');
```

Comment:
https://lodash.com/docs/4.17.15#pick

`pick` looks like it should only take an array.
```suggestion
        return _.pick(obj, ['duration', 'maxGradedAttemptsPerVersion', 'maxVersions', 'versionDelay']);
```

Reply:
I feel like variable arguments is sometimes documented as an array so it might not be wrong, I'll double check

Reply:
![image](https://user-images.githubusercontent.com/23609875/96818818-3f4cab00-13f0-11eb-8832-9890ac81a1e9.png)
Documentation has it as variable arguments in here
```js
const _ = require('lodash');

const obj = {
    a: 1,
    b: 2,
    c: 3,
    d: 4,
    e: 5
};

console.log(JSON.stringify(_.pick(obj, 'a','c','e')));
```
Results in:
```
{"a":1,"c":3,"e":5}
```